### DISCUSSION TITLE
Drop secondary indexes

### DISCUSSION THREAD

**Discussion by cezar-radan** [codenotary/immudb#1394] [Q&A] [answered]
It is possible to drop secondary indexes ? 

Theoretical should be possible since the indexes are technical elements and dropping an index will not have impact over the "real data" history.

Some time ago, when I start my project, I have created some indexes (zAdd) but the requirements evolved in time and now I don't need such indexes (they don't help me in any way). 
For me it is important to have a clean database schema.

**Comment by jeroiraz** [comment, marked as answer]
zAdd provides a higher level of abstraction but it's ultimately represented by a key-value pair. Thus it's not possible to directly remove them.

The physical deletion topic is high on the project's priority list
